plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataConfig.java

```diff
@@ -40,6 +43,8 @@
     // between performance and pushdown capabilities
     private int domainCompactionThreshold = 32;
 
+    private int insertBatchSize = 1000;
```

Comment:
This doesn't belong to JdbcMetadataConfig, since it's not consumed in JdbcMetadata layer

Reply:
I think this is mostly because there were no session properties contributed by BaseJdbcConfig.
Maybe this should change? WDYT?